rolling-shutter/medley/broker.go

```diff
@@ -0,0 +1,184 @@
+package medley
+
+import (
+	"sync"
+
+	"github.com/rs/zerolog/log"
```

Comment:
Don't really have an opinion on which logging package to use, but I think Schmir has looked into it. Any comments, @schmir-at-bb ?

Reply:
He introduced it somewhere else as well. I quite like this one